packages/autorest.python/test/azure/legacy/Expected/AcceptanceTests/AzureSpecials/azurespecialproperties/models/_models_py3.py

```diff
@@ -7,10 +7,16 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
+import sys
 from typing import Optional
 
 import msrest.serialization
 
+if sys.version_info >= (3, 8):
```

Comment:
We shouldn't add `Literal` import and definition here

Reply:
The constant is not in the argument list of `model.__init__` so the annotation is not used in the model files.

Fixed by changing at https://github.com/Azure/autorest.python/pull/1464/commits/4f7d4efc1cc7f0a304bc31177b7807cef969d731#diff-de62dad930cde83ace8497b5ee8e24a1f1b080f4d807247365c112c7b4a21b4bR143,